bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -77,6 +78,23 @@ contract("Bridge NFT", async function(accounts) {
     });
   });
 
+  const mintAndApprove = async (
+    token,
+    approveTo,
+    tokenId = 9,

```

Comment:
I'd maybe declare something like `const defaultTokenId = 9;` at the top of the test class, instead of using a raw 9

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/248/commits/f3a764baea46195d740e416c29e6bd5fcddb5710